PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -1139,8 +1139,10 @@ local function CreateCamera()
 					this.LastCameraTransform = nil
 				end
 			end
-			wait()
+			
 			setLookBehindCharacter()
+			wait()
```

Comment:
I think it was necessary because at the time of adding the character the engine had not actually rotated your character yet and you would spawn looking in the wrong direction.